src/commands/build.yml

```diff
@@ -54,6 +54,13 @@ parameters:
       Extra flags to pass to docker build. For examples, see
       https://docs.docker.com/engine/reference/commandline/build
 
+  cache_from:
+    type: string
+    default: ""
+    description: >
+      Comma separated list of containers to pull before build for --cache-from
+      https://docs.docker.com/engine/reference/commandline/build/
```

Comment:
Initially interpreted this as: this will do pulls for you, for the sake of --cache-from (but not the --cache-from itself).
Suggesting explicitly mentioning the steps. For example:

> Comma separated list of containers, images will first be pulled, then passed as the --cache-from build argument
> https://docs.docker.com/engine/reference/commandline/build/

Reply:
Good catch! I'll fix this up right away. And thank you for the review <3

Reply:
Just for completeness I checked the other arguments that pull offers (in case we want to expose) and I don't think any of these would be wanted (or for the verification step, even good) in the command.

```
Pull an image or a repository from a registry

Options:
  -a, --all-tags                Download all tagged images in the repository
      --disable-content-trust   Skip image verification (default true)
  -q, --quiet                   Suppress verbose output

```